sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ExtRefService.java

```diff
@@ -226,4 +228,23 @@ private static Optional<SclReportItem> configureControlBlockNetwork(ControlBlock
         return controlBlockAdapter.configureNetwork(appIdIterator.nextLong(), macAddressIterator.next(), settings.vlanId(), settings.vlanPriority(),
                 settings.minTime(), settings.maxTime());
     }
+
+    /**
+     * Remove ExtRef hich are fed by same Control Block
+     *
+     * @param tExtRefs List of ExtRef
+     * @return list ExtRefs without duplication
+     */
+    public static List<TExtRef> removeDuplicatedExtRefs(List<TExtRef> tExtRefs) {
```

Comment:
`removeDuplicatedExtRefs` seems like you are modifying the input List. I suggest `filterDuplicatedExtRefs` because you are creating a new `List`.